source/classes.tex

```diff
@@ -1387,12 +1387,11 @@
 
 \rSec2[class.nest]{Nested class declarations}%
 \indextext{definition!nested class}%
-\indextext{class local|see{local class}}%
-\indextext{class nested|see{nested class}}
+\indextext{class!nested|see{nested class}}
 
```

Comment:
Both "local class" and "nested class" have a bunch of subentries:

![subentries](http://eel.is/nestedlocal.png)

Should these be moved under "class, local" and "class, nested", respectively, as well, or stay where they are?